### DISCUSSION TITLE
Dump carte SD svp ?

### DISCUSSION THREAD

**Discussion by ghost** [marian-m12l/studio#264] [Q&A]
Ce matin j'ai fait mes transfert d'histoires habituel, après un retour de la lunii au SAV (molette cassée).
Tout s'est bien passé... Mais la lunii est briquée. "Error" avec l'image de la lunii et d'un ordi sur l'écran de la lunii.

J'ai essaye de ré-initialiser la lunii depuis l'appli officiel, mais sans succès.

Est-ce que quelqu'un pourrait partager un dump de la carte SD svp ?

**Comment by kpouget** [comment]
j'ai eu le meme problème, transfert d'une histoire (longue), Lunii débranchée (trop tot surement), `Error` et STUdio ne fonctionnant plus.

Ma solution 1: `rm .md` --> STUdio remarche, 0 histoires mais 23% de l'espace utilisé.
Ma solution 2: `rm -rf .content` --> espace utilisé à 0%, tout marche comme avant \o/

**Reply by ghost** [reply]
Merci :)
